js/animate_team.js

```diff
@@ -0,0 +1,20 @@
+var animate = anime({
+    targets: '.team__clip-path-container',
+    translateY: [
```

Comment:
Rozważ usunięcie całego sekcji kodu z translateY. Jak pierwszy raz wszedłem na stronę po zmianie, to poczułem się lekko zagubiony, bo zdjęcia odjechały a potem wróciły. Myślałem, że coś sie rozwaliło na stronie.

Reply:
Efekt odjeżdżających zdjęć zupełnie mi tutaj nie pasuje. Proponuję zostawić tylko opacity

Reply:
hehe no spoko, animacje to temat rzeka, chcialem cos prostego i widocznego zrobic, ale spoko mozemy zostac przy znikaniu